met-web/src/components/engagement/admin/create/authoring/AuthoringFeedback.tsx

```diff
@@ -0,0 +1,281 @@
+                    you would like your audience to do.
+                </EyebrowText>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
+                <label htmlFor="section_heading">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Section Heading <span style={{ fontWeight: 'normal' }}>(Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
+                        Your section heading should be descriptive, short and succinct.
+                    </EyebrowText>
+                    <TextField
+                        sx={{ backgroundColor: colors.surface.white }}
+                        value={sectionHeading}
+                        id="section_heading"
+                        counter
+                        maxLength={60}
+                        placeholder="Section heading message"
+                        onChange={(value) => {
+                            setSectionHeading(value);
+                        }}
+                    />
+                </label>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.gray[10] }} item>
+                <label htmlFor="body_copy">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Body Copy
+                        <span style={{ fontWeight: 'normal' }}> (Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
+                        You must either include links to your engagement feedback methods within this section’s body
+                        copy using the link button in the WSIWYG editor below, or, Call to Action Buttons in the
+                        following two fields.
+                    </EyebrowText>
+                    <RichTextArea
+                        spellCheck
+                        editorState={editorState}
+                        onEditorStateChange={handleEditorChange}
+                        handlePastedText={() => false}
+                        editorStyle={{
+                            height: '15em',
+                            padding: '1em',
+                            resize: 'vertical',
+                            backgroundColor: colors.surface.white,
+                        }}
+                        toolbar={toolbar}
+                    />
+                </label>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
+                <label htmlFor="survey_button_text">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Survey Call to Action Button <span style={{ fontWeight: 'normal' }}>(Optional)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
```

Comment:
My understanding of an "eyebrow" is that it's an additional piece of info that goes _above_ the "eye" - in this case the heading or label. I kind of feel this should be a label itself though

Reply:
I imported this eyebrow text component and used it because it resembled the type in the design, even though the name might not be 100% accurate. I could import it AS something else if you think it's a bit confusing.

Reply:
It's not necessarily confusing but it's important that we don't just code for looks but that our HTML is semantic and meaningful. In this case, I almost feel like both the metheader3 and eyebrowtext elements should be labels that describe the form field.

Reply:
I have removed all the MetHeader3 components and replaced them with MetLabel components that are imported as MetBigLabel. This means that they show up as <p> in the DOM instead of <h3>. Anything that is a real h3 heading is still an h3. I renamed EyebrowText to FormDescriptionText so that it's clear what its use is. This text is all already contained within a label element so it's part of the label, as is the form component itself.